packages/frontend/src/pages/Home/index.tsx

```diff
@@ -1,22 +1,16 @@
-import { Link, useLocation } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 import logo from '../../assets/images/Logo_long.png';
+import { Routes } from '../../constants/Routes';
 
 export const Home = () => {
-  const location = useLocation();
   return (
     <div>
       <header>
         <img src={logo} alt="logo" />
-        <p>This is unstyled homepage (authentication will either redirect to here or to access-denied page)</p>
+        <p>Sinua ei ole todennettu!</p>
```

Comment:
This sounds a bit off, but I can't really say what would be a better option 😄 Not a dealbreaker, we can modify it later.